apis/v1alpha1/httproute_types.go

```diff
@@ -495,9 +514,13 @@ type HTTPRouteForwardTo struct {
 	// ServiceName refers to the name of the Service to forward matched requests
 	// to. When specified, this takes the place of BackendRef. If both
 	// BackendRef and ServiceName are specified, ServiceName will be given
-	// precedence. If the referent cannot be found, controllers must set the
-	// "InvalidRoutes" status condition on any Gateway that includes this
-	// Route to true.
+	// precedence.
+	//
+	// If the referent cannot be found, the route must be dropped
+	// from the Gateway. The controller should raise the "ResolvedRefs"
+	// condition on the Gateway with the "DroppedRoutes" reason.
+	// The gateway status for this route should be updated with a
+	// condition that describes the error more specifically.
```

Comment:
If there were multiple forwardTos specified here, would it be safer to defer to any valid ones than to drop the route entirely?